.go-env.sh

```diff
@@ -4,12 +4,25 @@ set -Eeuo pipefail
 dir="$(dirname "$BASH_SOURCE")"
 dir="$(readlink -ve "$dir")"
 
-user="$(id -u):$(id -g)"
+src="$dir"
+dst="$dir"
+msys=
+if [ "$(uname -o)" = 'Msys' ];	then
+	msys=1
+	if command -v cygpath > /dev/null; then
+		src="$(cygpath.exe --windows "$dst")"
+	fi
+fi
+windowsContainers=
+if [ "$(docker version --format '{{ .Server.Os }}')" = 'windows' ]; then
```

Comment:
```suggestion
serverOs="$(docker version --format '{{ .Server.Os }}')"
if [ "$serverOs" = 'windows' ]; then
```
(this one, however, _will_ be a useful failure because `docker version` failing means our daemon probably isn't working so we can bail eariler)